tools/licenses/lib/patterns.dart

```diff
@@ -118,6 +118,7 @@ final List<RegExp> copyrightStatementPatterns = <RegExp>[
   RegExp(r'^(?:[^ ]+ )?Modifications:$', caseSensitive: false),
   RegExp(r'^ *Modifications for', caseSensitive: false),
   RegExp(r'^ *Modifications of', caseSensitive: false),
+  RegExp(r'^Modifications Copyright \(C\) .+', caseSensitive: false),
```

Comment:
Needed for: https://github.com/search?q=repo%3AKhronosGroup%2FVulkan-ValidationLayers+%22Modifications+Copyright+%28C%29+2022+RasterGrid+Kft.%22&type=code